Sage/carbon
# docs(storybook): remove design system directory

### Proposed behaviour

Remove the `Design System` directory from storybook. The components will be nested under the storybook root instead.

### Current behaviour

The `Design System` directory was used as a market to know which components were aligned with the previous Design System. Now there is a new `Design System` and a dedicated team to align these components, this directory serves no purpose. All components should be part of the `Design System` or removed from `carbon-react`.

### Checklist




#### QA


### Additional context
The `cypress/fixtures/designSystem` directory still exists in the code base. This can be moved. I chose not to do that in this PR to make the review process easier.

### Testing instructions

View the new version of storybook, by clicking `details` on the `Storybook Publish` check below.
